### PR TITLE
[4.1] Upmerge from 3.10

### PR DESCRIPTION
**Pull request by bembelimen**
Pull Request for Issue # .

### CODE REVIEW

File: layouts/joomla/html/image.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * @package     Joomla.Site
  * @subpackage  Layout
  *
- * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
```
Comment (brianteeman) [inline layouts/joomla/html/image.php]: copyright date should not change

File: administrator/modules/mod_stats_admin/mod_stats_admin.php [hunk 2]
```diff
@@ -9,13 +9,12 @@
 
 defined('_JEXEC') or die;
 
-use Joomla\CMS\Factory;
-use Joomla\CMS\Helper\ModuleHelper;
-use Joomla\Database\DatabaseInterface;
-use Joomla\Module\StatsAdmin\Administrator\Helper\StatsAdminHelper;
+// Include the mod_stats functions only once
```
Comment (brianteeman) [inline administrator/modules/mod_stats_admin/mod_stats_admin.php]: this entire file change looks wrong

Comment (bembelimen) [reply administrator/modules/mod_stats_admin/mod_stats_admin.php]: You're right, thx

File: layouts/joomla/html/image.php (hunk 1 above)
Comment (bembelimen) [reply layouts/joomla/html/image.php]: Thx

File: layouts/joomla/html/image.php [hunk 3]
```diff
@@ -19,9 +19,12 @@
 use Joomla\CMS\HTML\HTMLHelper;
 use Joomla\Utilities\ArrayHelper;
 
-$img = HTMLHelper::_('cleanImageURL', $displayData['src']);
+if (isset($displayData['src']))
+{
+	$displayData['src'] = $this->escape($displayData['src']);
```
Comment (dgrammatiko) [inline layouts/joomla/html/image.php:24]: This is wrong, the `$displayData['src']` should be the `$img->src` from the `$img = HTMLHelper::_('cleanImageURL', $displayData['src']);`
basically something like
```php
if (isset($displayData['src']))
{
	$img = HTMLHelper::_('cleanImageURL', $displayData['src']);
	$displayData['src'] = $this->escape($img->src);

	if ($img->attributes['width'] > 0 && $img->attributes['height'] > 0)
	{
		$displayData['width']  = $img->attributes['width'];
		$displayData['height'] = $img->attributes['height'];
		if (empty($displayData['loading']))
		{
			$displayData['loading'] = 'lazy';
		}
	}
}
```

Comment (dgrammatiko) [reply layouts/joomla/html/image.php:24]: @bembelimen this needs to be fixed, the code that you merged is not right

Comment (zero-24) [reply layouts/joomla/html/image.php:24]: Has been fixed by Benjamin here already: 

https://github.com/joomla/joomla-cms/commits/4.1-dev/layouts/joomla/html/image.php

Comment (dgrammatiko) [reply layouts/joomla/html/image.php:24]: Nice

**Pull request merged**